### DISCUSSION TITLE
Is there a moose application for benchmarking and comparing results?

### DISCUSSION THREAD

**Discussion by lbgpublic** [idaholab/moose#18922] [Q&A General] [answered]
Hi

Is there a shared moose application that people use for benchmarking and comparing results?  In particular I would be interested to compare results when using mpi to run moose in parallel across many nodes.

Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

For benchmarking performance (scaling plots), there's a script that was merged in this PR #18005 that you can adapt quite easily
For comparing results, you can load previous results using a SolutionUserObject and then there's plenty of postprocessors and auxkernels to compute differences between two solutions. 

Guillaume
